src/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_infer_request/batched_tensors.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/ov_infer_request/batched_tensors.hpp"
+
+using namespace ov::test::behavior;
+
+namespace {
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests, OVInferRequestBatchedTests,
```

Comment:
could you also add one test case the same as `SetInputTensorsBase` but when we load dynamic model and execute 2 infer requests with different shapes